src/containers/PipelineRun/PipelineRun.js

```diff
@@ -97,6 +97,21 @@ export /* istanbul ignore next */ class PipelineRunContainer extends Component {
 
   render() {
     const { error, intl, match, pipelineRun, tasks, taskRuns } = this.props;
+
+    if (!this.props.pipelineRun) {
+      return <PipelineRun error="PipelineRun not found" loading={false} />;
+    }
+
+    if (!this.props.pipelineRun.status) {
+      pipelineRun.status = 'unknown';
```

Comment:
didn't spot this earlier but looks like something fishy going on here?

`status = 'unknown'`
then immediately below
`status = somethingElse`

was this supposed to be updating `reason` or perhaps a `status` field somewhere else?

Reply:
Hmmmmmm, so what's there is intentional - if there's no status, set a status with no taskruns for that field

If there was actually a status anyway we enter the .status.taskRuns check and we provide an empty array of taskruns there

Reply:
within this single conditional block you assign a value to status, then immediately reassign it

so the `status= 'unknown'` is redundant

Reply:
True, addressed and pushed, oops